newIDE/app/src/MainFrame/Preferences/PreferencesDialog.js

```diff
@@ -246,6 +247,14 @@ const PreferencesDialog = ({ i18n, onClose }: Props) => {
               }
             />
           </Line>
+          <Line>
+            <Toggle
+              onToggle={(e, check) => setIsMenubarHiddenInPreview(check)}
+              toggled={values.isMenubarHiddenInPreview}
+              labelPosition="right"
+              label={<Trans>Hide menubar in preview window</Trans>}
```

Comment:
`Hide the menu in the preview window`

Reply:
(or "Hide the menu bar in the preview window")
https://en.wikipedia.org/wiki/Menu_bar#:~:text=A%20menu%20bar%20is%20a,displaying%20help%20documentation%20or%20manuals.